cluster-autoscaler/main.go

```diff
@@ -368,6 +376,36 @@ func getKubeConfig() *rest.Config {
 	return kubeConfig
 }
 
+func getSchedulerConfig(path string) *schedconfig.KubeSchedulerConfiguration {
+	data, err := os.ReadFile(path)
+
+	defaultFallback := "(falling back to default scheduler config)"
+
+	if err != nil {
+		klog.Errorf("couldn't load scheduler config %s: %v", defaultFallback, err)
+		return &schedconfig.KubeSchedulerConfiguration{}
+	}
+
+	obj, gvk, err := schedscheme.Codecs.UniversalDecoder().Decode(data, nil, nil)
```

Comment:
This line creates a default scheduler config if none is specified. 
TODO: return output of this call as default. If it errors, fall back to the old behavior of getting a default config. 

Not doing the TODO above leads to a problem where the scheduler config ends up as empty and it shows up in the error log like this:
```
suraj@suraj:~/sandbox/autoscaler/cluster-autoscaler$ ./cluster-autoscaler --kubeconfig=/home/suraj/.kube/config --namespace=default
I0428 19:48:18.545718 1865631 leaderelection.go:245] attempting to acquire leader lease default/cluster-autoscaler...
I0428 19:48:40.443736 1865631 leaderelection.go:255] successfully acquired lease default/cluster-autoscaler
E0428 19:48:40.444001 1865631 main.go:385] couldn't load scheduler config (falling back to default scheduler config): open : no such file or directory
F0428 19:48:40.446465 1865631 main.go:504] Failed to create autoscaler: unexpected scheduler config: expected one scheduler profile only (found 0 profiles)
```